.github/workflows/newer-java.yml

```diff
@@ -17,7 +27,6 @@ jobs:
       max-parallel: 1
       matrix:
         include:
-          - java-version: 17
```

Comment:
one more ?!

Reply:
Let's leave that - it'll go away w/ #7783 anyway